tests/cmd/scan/test_path.py

```diff
@@ -42,6 +41,7 @@ def test_scan_file(self, cli_fs_runner, verbose):
             result = cli_fs_runner.invoke(cli, ["-v", "secret", "scan", "path", "file"])
         else:
             result = cli_fs_runner.invoke(cli, ["secret", "scan", "path", "file"])
+        assert result.exit_code == 0, result.output
```

Comment:
nit: since we use this assertion a lot, we might want to extract a method `assert_successful_scan()` and a method `assert_failed_scan(exception)`
```
def assert_successful_scan():
    assert result.exit_code == 0, result.output
    assert not result.exception
    ...
```

Reply:
Here it is: https://github.com/GitGuardian/ggshield/issues/253